src/test/java/org/mastodon/mamut/feature/spot/ellipsoid/AbstractEllipsoidFeatureTest.java

```diff
@@ -0,0 +1,51 @@
+package org.mastodon.mamut.feature.spot.ellipsoid;
+
+import org.mastodon.feature.Feature;
+import org.mastodon.feature.FeatureProjection;
+import org.mastodon.feature.FeatureProjectionKey;
+import org.mastodon.feature.FeatureProjectionSpec;
+import org.mastodon.mamut.model.Model;
+import org.mastodon.mamut.model.ModelGraph;
+import org.mastodon.mamut.model.Spot;
+
+import java.io.IOException;
+
+public abstract class AbstractEllipsoidFeatureTest
```

Comment:
This does not need to be an abstract class. I think you a better off with a normal "SingleEllipsoidExampleGraph" class.

Reply:
I would like to keep this class for more EllipsoidFeatureTests to come. I thought about an extra SingleEllipsoidExampleGraph, but it does not seem to simplify things at the moment